src/server/user_handler.cpp

```diff
@@ -49,3 +50,47 @@ std::string user_handler::create_salt(int length) {
 
 	return  ss.str();
 }
+
+// TODO - This really should be a common function but elsewhere it's a virtual member and I don't feel like fixing the mess tonight.
+namespace {
+	const std::string itoa64 = "./0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" ;
```

Comment:
This is also a problem in the original code, but this isn't logically a string. It's an array of characters. Therefore, please change its type to `const char[]`.

Reply:
Well, make it a `const std::array::<char, 64>` then.

Reply:
won't fix. see comment about the TODO